spec/controllers/comments_controller_spec.rb

```diff
@@ -15,19 +15,23 @@
       end
 
       it "sends a comment email to approvers and observers" do
+        ENV["DISABLE_EMAIL"] = nil
```

Comment:
This is totally something which should be refactored. I'll look into how it'd work with an rspec tag.